src/bootstrap4-floating-label.scss

```diff
@@ -53,6 +53,14 @@ $label-top-position-md: #{- $label-height-md + $label-height-md / 2.5}; // -24px
 $label-height-lg: $font-size-lg * $input-line-height-lg; // 30px
 $label-top-position-lg: #{- $label-height-lg + $label-height-lg / 2.5}; // -30px + (30px / 2) = -15px
 
+// Enable RTL mode
+html[dir="rtl"] {
```

Comment:
First thing first we need to update example/index.html and test everything is OK:
```HTML
      <h2>Support for "right to left" languages</h2>
      <bdo dir="rtl">
        <div class="form-group floating-label">
          <input id="input-rtl" class="form-control" placeholder="input RTL" />
          <label for="input-rtl">input RTL</label>
        </div>
        <div class="form-group floating-label">
          <select class="form-control" id="select-rtl">
            <option>1</option>
            <option>2</option>
            <option>3</option>
            <option>4</option>
            <option>5</option>
          </select>
          <label for="select-rtl">select RTL</label>
        </div>
      </bdo>
```

What we want to achieve:
![image](https://user-images.githubusercontent.com/643434/66598091-8f1f5200-eba0-11e9-9203-1a99f8097e4a.png)

So do we need to specify **html**[dir=...]? No as we want to be able to use `<bdo dir="rtl">` (or `<html dir="rtl">` or `<body dir="rtl">`).

btw nice trick: %label-rtl is not define if no rtl and thus `@extend %label-rtl` does nothing 👍